### ISSUE TITLE
Implement backup and restore of just archived MURs

### ISSUE THREAD

**Issue by vrajmohan** [18F/openFEC#2810]
Although we can backup and restore _all_ the legal documents quite easily, we need to figure out a way to do this for just archived MURs. Whenever there is a schema change in Elasticsearch, all legal documents need to be reloaded. As this takes an excessively long time for archived MURs (over 10 hours, and due to errors it can take 3-4 days), we need to design a way to backup and restore just archived MURs.

See below for design

- [x] Load test `dev` using locust to make sure there's no performance hit - do a baseline then a manual deploy, reindex and rload current and archived murs, test again
Result: No decrease in speed - actually, there was a slight increase. [Results are here](https://docs.google.com/spreadsheets/d/1i8T9V5FnKApmsf2iOnLiHA1_3q1r3xmKlmmoT7hJ8x0/edit#gid=1519164865)
- [X] Move the existing archived MURs from `docs` to `archived_murs` using the bulk reindex command (this has to be done once only) on `dev`

Created new issue #2860 for these:
   - Backup `prod` and reload `stage` after the RSB-4 release is cut (1/24)
   - Backup `prod` and reload after deploy

 **Steps to move archived MURs:**
```
./manage.py create_archived_murs_index
./manage.py move_archived_murs
./manage.py refresh_current_legal_docs_zero_downtime
```

**Comment by vrajmohan** [comment]
As far as I can tell, `Elasticsearch` snapshot and restore works only with the entire contents of indexes - you can select individual indexes to  snapshot, but not specific documents within those indexes.

We could create a second index, say, `archived_murs_index` which would only store archived MURs. The existing archived MURs would be _moved_ from `docs` to `archived_murs_index` using the bulk `reindex` command (this has to be done once only). In addition, `load_archived_MURs` would be modified to load archived MURs into `archived_murs_index`. This feature may never be executed as archived MURs are rarely, if ever, updated.

`archived_murs_index` would be independently snapshotted and restored.

Utilizing this second index in all document searches can then be implemented in 2 ways:

|Option|Pros|Cons|
|-------|----|------|
|The `docs_search`  alias would point to both `docs` and `archived_murs_index`. |Less effort to implement|Performance issues when Elasticsearch has to scan multiple indexes. This needs to be tested to see if it is indeed a problem|
|Use bulk `reindex` command to manually copy over archived MURs alone from `archived_murs_index`| No performance implications - the end result will be equivalent to what we have now | More effort to code, we would have to hide all the mess behind a management command.|

For reference, the `reindex` command to use would be similar to https://github.com/18F/openFEC/blob/develop/webservices/legal_docs/index_management.py#L425

**Comment by lbeaufort** [comment]
Addressed by #2860: Move archived MURs to their own index on staging and production environments
